pyfstat/make_sfts.py

```diff
@@ -484,6 +377,40 @@ def calculate_fmin_Band(self):
                 "Generating SFTs with fmin={}, Band={}".format(self.fmin, self.Band)
             )
 
+    def get_single_config_line(self, i):
+        config_line = "[TS{}]\n".format(i)
+        config_line += "\n".join(
+            [
+                "{} = {{{}}}".format(key, self.signal_formats[key]).format(
+                    self.signal_parameters[key]
+                )
+                for key in self.signal_parameters.keys()
+            ]
+        )
+        config_line += "\n"
+
+        return config_line
+
+    def make_cff(self, verbose=False):
+        """
+        Generates a .cff file
+
+        """
+
+        content = self.get_single_config_line(0)
+
+        if verbose:
+            logging.info(
+                "Writing the following injection parameters"
+                " to config file {:s}:".format(self.config_file_name)
+            )
+            logging.info(content)
+
+        if self.check_if_cff_file_needs_rewritting(content):
+            config_file = open(self.config_file_name, "w+")
+            config_file.write(content)
+            config_file.close()
```

Comment:
seems we lost some verbosity option here, but it didn't seem very informative, so I guess that's ok

Reply:
Ups, this was not on purpose, I'll add it: It's useful to have the actual parameters after parsing on display (*ejm*)

Reply:
Actually... we did not loose any verbosity, did we?